src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -371,7 +371,10 @@ private void UpdateInputScope(InputScope inputScope)
 
 		partial void OnTextWrappingChangedPartial(DependencyPropertyChangedEventArgs e)
 		{
-			//TODO : see bug #8178
+			if (_textBoxView != null && e.NewValue is TextWrapping textWrapping && e.OldValue is TextWrapping oldTextWrapping && oldTextWrapping != textWrapping)
```

Comment:
It's not a big deal, but just to you know, property changed events are only raised when the value has really changed:

https://github.com/unoplatform/uno/blob/d403a71fe22c32f4a3531b202e9a255c6261906c/src/Uno.UI/UI/Xaml/DependencyObjectStore.cs#L1914

 so there's no need to compare it again here:
```suggestion
			if (_textBoxView != null && e.NewValue is TextWrapping textWrapping && e.OldValue is TextWrapping oldTextWrapping)
```